rancher-config/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+services:
+  pharma-etl:
+    image: nhsuk/pharmacy-data-etl:${DOCKER_IMAGE_TAG}
+    environment:
+      AZURE_STORAGE_CONNECTION_STRING: ${AZURE_STORAGE_CONNECTION_STRING}
+      CONTAINER_NAME: ${CONTAINER_NAME}
+      SYNDICATION_API_KEY: ${SYNDICATION_API_KEY}
+      ETL_SCHEDULE: ${ETL_SCHEDULE}
+      NODE_ENV: production
+    stdin_open: true
```

Comment:
These settings were obtained by deploying the image manually to Rancher, then using the generated projects docker-compose.yml in the rancher-config.  Seems to work OK without them, will remove the options.